Code-review/index.html

```diff
@@ -0,0 +1,197 @@
+                    <br>
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            
+            
+</div>
+<div>
+    <h1>Youtube video page</h1>
```

Comment:
I think you could have used a paragraph for this case instead of a heading (h1). headings usually have a block of text below them but this one does not. I kindly suggest you change that. not so?🤔